src/_includes/common-form-fields.html

```diff
@@ -0,0 +1,134 @@
+{# ==========================================================================
+
+   common_form_fields.state()
+
+   ==========================================================================
+
+   Description:
+
+   Returns a select box with all 50 states and DC.
+
+   legend: Name of the field.
+
+   size: The size of the fields. Defaults to 1-3.
+          Possible options: 1, 1-2, 1-3
+
+   required: Whether the field is required. Defaults to false.
+
+   disabled: Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% macro state(size='1-3', required=false, disabled=false) %}
+  {% from 'macros/form.html' import select as select %}
+  {{ select('State',
+            'Select state',
+            [ 'Alabama',
```

Comment:
Yep. Right now it's using slugify, but does it make sense to pass through an object instead of an array so that we could have control over what we use for value and text or should we keep it simple? I was a little torn on this one.